colossalai/nn/layer/parallel_1d/layers.py

```diff
@@ -7,25 +7,44 @@
 
 import torch
 import torch.nn.functional as F
+from apex.normalization import FusedLayerNorm
```

Comment:
`apex` is not a necessary dependency for Colossal-AI, importing from apex here will raise exception.